db/migrations/20190411132245_create_all_ilks_query.sql

```diff
@@ -1,127 +1,17 @@
 -- +goose Up
 
 -- Function returning state for all ilks as of the given block height
-CREATE FUNCTION api.all_ilks(block_height BIGINT DEFAULT api.max_block(), max_results INTEGER DEFAULT NULL,
-                             result_offset INTEGER DEFAULT 0)
-    RETURNS SETOF api.ilk_state
+CREATE FUNCTION api.all_ilks(block_height BIGINT DEFAULT api.max_block())
```

Comment:
I removed the pagination stuff from this query. It can only return as many results as there are collateral-types (currently 2), so pagination felt like overkill haha

Reply:
Dig it. Has me wondering whether there might be a better name for this query 🤔